app/controllers/group_assignment_invitations_controller.rb

```diff
@@ -67,6 +67,8 @@ def create_group_assignment_repo(selected_group: group, new_group_title: nil)
     users_group_assignment_repo = invitation.redeem_for(current_user, selected_group, new_group_title)
 
     if users_group_assignment_repo.present?
+      options = { subscribed: true }
+      current_user.github_client.update_subscription(users_group_assignment_repo.github_repository.full_name, options)
```

Comment:
So for actions on GitHub resources, we have a bunch of classes that inherit from `GitHubResource` where we put this logic. We're subscribing a user to a repository, and we need to be sure we're using the user's token, so I think this logic belongs in `GitHubUser`. `GitHubUser#accept_repository_invitation` is a good example of how this method could look.